src/newton_raphson.toit

```diff
@@ -63,13 +63,13 @@ example2:
   // defined at 0.
   // We define the goal as 10.0 because we have 10 on the right hand side
   // instead of 0.0, which is the default.
-  solution := newton_raphson.solve --goal=10.0 --initial=1.0 --function=function --derivative=derivative
+  solution := newton-raphson.solve --goal=10.0 --initial=1.0 --function=function --derivative=derivative
 
   // Prints 2.0997856714017091306 because this is the solution of ln(x) + x^3 = 10.0
   print solution
 ```
 */
-solve --initial/num=0.0 --goal/num=0.0 --max-iterations=20 --precision=1e9 [--function] [--derivative] [--no-convergence] -> float:
+solve --initial/num=0.0 --goal/num=0.0 --max-iterations=20 --precision=1e9 [--function] [--derivative] [--no_convergence] -> float:
```

Comment:
What happened here?

Reply:
this is a bug in 'kebabify'.
We are not allowed to rename '--no_convergence' to '--no-convergence' as named parameters are not allowed to start with '--no-'.
For now I just renamed it back, but I will write a PR that deprecates that parameter and replace it with a differently named one.